fastlane/env/project.env-example

```diff
@@ -1,8 +1,6 @@
 INT_EXPORT_TEAM_ID=<Team id for internal distribution>
 EXT_EXPORT_TEAM_ID=<Team id for public distribution>
 
-APPCENTER_PUBLIC_ID=<AppCenter Public Id>
-
```

Comment:
Similarly, once this PR lands, we might want to remove that env var value from the actual `project.env` file in `~/.mobile-secrets` ([see `.configure` setup](https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/.configure#L6-L10)), as it doesn't seem used anymore either.

Reply:
This has been tracked in https://github.com/Automattic/apps-infra-plans/issues/219.